src/main/java/org/fcrepo/camel/processor/ProcessorUtils.java

```diff
@@ -115,5 +126,24 @@ public static String insertData(final String serializedGraph, final String named
         query.append("}");
         return query.toString();
     }
+
+    /**
+     * Tokenize a property placeholder value
+     *
+     * @param context the camel context
+     * @param property the property name
```

Comment:
Although it'll be apparent by looking at the tests, I think the Javadoc would be a bit more clear if it said "the property placeholder", or "the property name surrounded by double braces" rather than "property name".  

That's my only comment; looks great!